README.md

```diff
@@ -221,6 +221,7 @@ A sample configuration file with all options is available [here](https://github.
     * `"check-constructor"` enforces explicit visibility on constructors (can only be public)
 * `member-ordering` enforces member ordering. The first option should be an object with an `order` key.
    Values for `order` can be `fields-first`, `statics-first`, `instance-sandwich`, or a custom order.
+* `mergeable-namespace` disallows mergeable namespaces in the same file.
```

Comment:
sorry, I should've clarified on the original issue -- this rule should be called __`no-mergeable-namespace`__ because it lints against the existence of certain syntax.